src/app/applyContextMenu.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Menu, MenuItem } from 'electron'
+
+/**
+ * Enable context menu by right click
+ *
+ * @param {import('electron').BrowserWindow} browserWindow - Browser window
+ */
+export function applyContextMenu(browserWindow) {
+
+	browserWindow.webContents.on('context-menu', async (event, params) => {
+		event.preventDefault()
+		// browserWindow.webContents.send('open-context-menu', params)
```

Comment:
Not sure, if we should prevent default behavior by default. What is the default behavior here?

Is `send('open-context-menu', params)` for a custom context menu in the app by a custom event?

Reply:
Seems that in-browser events are trigerred first. So, if we would do something like that:
```js
  document.body.addEventListener('contextmenu', (event) => {
    console.log('spreed repository event')
    event.preventDefault()
  })
```
It will prevent Electron event from calling. So i will remove it from here, but keep in mind the order